### PR TITLE
Change ".." punctuation for "..." in editor strings

### PR DESCRIPTION
**Pull request by Calinou**
The correct punctuation for continuation is "...", not "..".

### CODE REVIEW

**Comment by capnm** [comment]
It would probably be better to use the Unicode character ...

File: thirdparty/tinyexr/tinyexr.h [hunk 1]
```diff
@@ -563,7 +563,7 @@ namespace miniz {
        - Eliminated a bunch of warnings when compiling with GCC 32-bit/64.
        - Ran all examples, miniz.c, and tinfl.c through MSVC 2008's /analyze
    (static analysis) option and fixed all warnings (except for the silly
-        "Use of the comma-operator in a tested expression.." analysis warning,
+        "Use of the comma-operator in a tested expression..." analysis warning,
```
Comment (Noshyaar) [inline thirdparty/tinyexr/tinyexr.h]: This file is in `thirdparty/`. 🤔

**Comment by Calinou** [comment]
> It would probably be better to use the Unicode character ...

I'm not sure how well it would play with translated strings (since there would be Unicode characters in source strings). If this is known to work well, I'll change it for `…` instead of `...`.

**Comment by akien-mga** [comment]
I don't know about this change, `..` seems to be @reduz's own way to notify submenus, it kind of makes sense eventually :)

**Comment by reduz** [comment]
From what i checked, all apps use ..., my fault for using ..
I suppose we can merge, though this needs to be fixed (conflicts)

**Comment by Calinou** [comment]
I updated the pull request, please check it again :smiley:

**Comment by akien-mga** [comment]
This will break all translations for those strings... Well I'm in for some sed magic next time I sync with Weblate I guess.

**Pull request merged**